src/apis/api_v1.py

```
@@ -0,0 +1,220 @@
+        response = ai_platform_src.poll_thread_result(thread_id=thread_id)
+
+        results.append(response)
+
+    try:
+
+        if webhook_config:
+            webhook = CustomWebhook(WebhookConfig(**webhook_config))
+            logger.info(
+                f"Posting results to the webhook configured at {webhook.config.endpoint}"
+            )
+            res = webhook.post_result({"results": results, "session_id": fa.session.id})
```

Ruff (0.11.9)

Undefined name `fa`

(F821)